freshmaker/handlers/botas/botas_shipped_advisory.py

```diff
@@ -115,6 +154,27 @@ def handle(self, event):
         bundles_nvrs = self._filter_bundles_by_pinned_related_images(
             auto_rebuild_nvrs)
 
+        # Now we have the bundle NVRs needs to be rebuilt, for each bundle NVR,
+        # construct the data of related images pullspec overrides
+        pullspec_overrides_by_nvr = {}
```

Comment:
We need to end up with the same format that OSBS uses in the Brew build extra field of `image.operator_manifests.related_images.pullspecs`. You can see an example of this with the Brew build with the ID of 1464389.

It also needs to include the pullspecs that Freshmaker did not replace. So in summary, use the dictionary of `image.operator_manifests.related_images.pullspecs` as a starting point. Then update the references of the `new` fields for pullspecs that were updated to be the rebuilt digest instead of the vulnerable one.